### ISSUE TITLE
Mentors editing their team submissions

### ISSUE THREAD

**Issue by stenington** [Iridescent-CM/technovation-app#1383]
QA

- Login as a mentor on a team (or create one for your mentor)
- Verify that you can start and edit a submission for your teams

Whatever functionality students currently have, mentors should have, be aware that many pieces of the submission are still a work in progress, so if the you find that the mentor has the same things that are missing / buggy as you find when working on submissions as a student, that is likely okay. Refer to the To-Do / In Progress lists to see if related work is planned before reporting something that's missing or buggy

**Comment by monicagragg** [comment]
Hi @joemsak, I logged into my mentor account to QA.

I was able to:
create the name of the app
write a description
add a demo video
add a pitch video
choose a development platform 
upload six screenshots (and when I tried to load more the red X appeared on the image

Love:
"Your progress has been saved. Keep up the good work!"
Competition qualification with the reasons why and how we can fix it.
"See how it looks to the judges"
"Change your app description"

Thoughts
Description - For our international audience, we should put a word cap to support, "only write a few sentences." And maybe reword to, "In 100 words or less..."
The source code says zip file or aia. I was able to add a jpeg. Not sure if we want to block other files.

**Comment by joemsak** [comment]
Thanks for your QA, @monicagragg! I have updated the description piece to count up to 100 words and disable the save button until the user corrects their input.

As for the file uploading validation, since we use the same uploader for different upload types, it's pretty complicated to limit the types. Last year it wasn't an issue, students uploaded the correct types.

We also provide a link for judges to email support if there is any problem with the links, and then we notify the students, giving them a chance to fix it.

**Comment by madhavitechnovation** [comment]
This card looks good. I was able to upload submission for my team.
